api_tests/lib/cnd_runner.ts

```diff
@@ -25,7 +25,7 @@ export class CndRunner {
                 continue;
             }
 
-            this.runningNodes[name] = await spawn(
```

Comment:
:question: Is this part of this PR?

Reply:
Yes, there is a dedicated commit for it :)

Reply:
Oops, missed that.